### ISSUE TITLE
Error when using library expansion: Cannot read property 'definition' of null

### ISSUE THREAD

**Issue by agustinlg** [raml-org/raml-js-parser-2#587]
When using library expansion with this [RAML file](https://github.com/raml-org/raml-js-parser-2/files/678078/petstore2.yaml.txt), and error is thrown:

```
TypeError: Cannot read property 'definition' of null
      at node_modules/raml-1-parser/src/raml1/ast.core/builder.ts:895:65
      at Array.forEach (native)
      at patchTypeWithFacets (node_modules/raml-1-parser/src/raml1/ast.core/builder.ts:882:29)
      at desc1 (node_modules/raml-1-parser/src/raml1/ast.core/builder.ts:925:27)
      at descriminate (node_modules/raml-1-parser/src/raml1/ast.core/builder.ts:998:23)
      at doDescrimination (node_modules/raml-1-parser/src/raml1/ast.core/builder.ts:977:12)
      at BasicNodeBuilder.process (node_modules/raml-1-parser/src/raml1/ast.core/builder.ts:115:29)
      at ASTNodeImpl.children (node_modules/raml-1-parser/src/raml1/highLevelImpl.ts:1722:42)
      at ASTNodeImpl.attrs (node_modules/raml-1-parser/src/raml1/highLevelImpl.ts:1938:39)
      at ASTNodeImpl.attributes (node_modules/raml-1-parser/src/raml1/highLevelImpl.ts:1934:30)
      at ReferencePatcher.patchReferences (node_modules/raml-1-parser/src/raml1/ast.core/referencePatcher.ts:66:35)
      at ReferencePatcher.patchType (node_modules/raml-1-parser/src/raml1/ast.core/referencePatcher.ts:322:34)
      at ReferencePatcher.patchReferences (node_modules/raml-1-parser/src/raml1/ast.core/referencePatcher.ts:88:18)
      at ReferencePatcher.patchReferences (node_modules/raml-1-parser/src/raml1/ast.core/referencePatcher.ts:95:18)
      at ReferencePatcher.patchReferences (node_modules/raml-1-parser/src/raml1/ast.core/referencePatcher.ts:95:18)
      at ReferencePatcher.patchReferences (node_modules/raml-1-parser/src/raml1/ast.core/referencePatcher.ts:95:18)
      at ReferencePatcher.process (node_modules/raml-1-parser/src/raml1/ast.core/referencePatcher.ts:38:14)
      at TraitsAndResourceTypesExpander.expandHighLevelNode (node_modules/raml-1-parser/src/raml1/ast.core/expander.ts:153:16)
      at LibraryExpander.expandLibraries (node_modules/raml-1-parser/src/raml1/ast.core/expander.ts:477:48)
      at Object.expandLibraries (node_modules/raml-1-parser/src/raml1/ast.core/expander.ts:38:34)
      at expandLibraries (node_modules/raml-1-parser/src/raml1/wrapped-ast/wrapperHelper.ts:96:21)
      at Object.expandSpec (node_modules/raml-1-parser/src/raml1/wrapped-ast/wrapperHelper.ts:73:16)
      at ApiImpl.expand (node_modules/raml-1-parser/src/raml1/artifacts/raml10parser.ts:6860:27)
      at lib/importers/baseraml.js:546:29
      at RAML10Importer.loadData (lib/importers/baseraml.js:540:10)
      at Auto._parseRAML (lib/importers/auto.js:65:8)
      at lib/importers/auto.js:146:18
      at Auto.loadData (lib/importers/auto.js:138:10)
      at lib/exporters/exporter.js:48:15
      at lib/exporters/exporter.js:44:47
      at lib/exporters/exporter.js:39:8
      at RAML10Exporter._export (lib/exporters/exporter.js:38:10)
      at lib/converter.js:52:9
      at Converter.convert (lib/converter.js:49:12)
      at lib/converter.js:23:14
```

**Comment by ddenisenko** [comment]
Fixed. Fix should be published in 1.1.12

**Comment by agustinlg** [comment]
Verified ok
